tripal_chado/src/Task/ChadoPreparer.php

```diff
@@ -895,164 +900,2350 @@ private function getVocabulary($name) {
   }
 
   /**
-   * Loads ontologies necessary for creation of default Tripal content types.
+   * Gets a term by its idSpace and accession
+   *
+   * @param string $idSpace
+   *   The Id space name for the term.
+   * @param string $accession
+   *   The accession for the term.
+   * @return TripalTerm|NULL
+   *   A tripal term object.
    */
-  protected function loadOntologies() {
-
-    $ontologies = [];
+  private function getTerm($idSpace, $accession, $vocabulary = NULL) {
+    $id = $this->getIdSpace($idSpace);
+    if ($vocabulary) {
+      $id->setDefaultVocabulary($vocabulary);
```

Comment:
I don't believe this should be here... `setDefaultVocabulary()` is meant to indicate the original vocabulary which created the ID Space and is part of the network to support borrowed terms. We may have made an assumption that ID Spaces are always unique in which case the vocabulary is not needed at all here. At most you may want to check that the vocabulary passed in is either the default or an accessory vocabulary for the term but that doesn't seem worth the performance hit 🤷‍♀️